source/_integrations/yale.markdown

```diff
@@ -0,0 +1,167 @@
+---
+title: Yale
+description: Instructions on how to integrate your Yale devices into Home Assistant.
+ha_category:
+  - Binary sensor
+  - Button
+  - Camera
+  - Doorbell
+  - Event
+  - Lock
+  - Sensor
+ha_release: 2024.9
+ha_iot_class: Cloud Push
+ha_config_flow: true
+ha_codeowners:
+  - '@bdraco'
+ha_domain: yale
+ha_dhcp: true
+ha_platforms:
+  - binary_sensor
+  - button
+  - camera
+  - diagnostics
+  - event
+  - lock
+  - sensor
+ha_integration_type: integration
+---
+
+The **Yale** integration allows you to integrate your [Yale](https://yale.com/) and some Yale Access devices in Home Assistant.
```

Comment:
I don't think that URL goes to the right company.

Reply:
e970e3fcf30d6bdf95a90e60a002641c27498628